src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,13 +28,26 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        this.sequenceNumber = nextSequenceNumber;
+        incrementSeqNo();
```

Comment:
what happens to SLAP when you use a method to increment the nextSequenceNumber? Is it the same abstract level?